### DISCUSSION TITLE
Idea: Adding Custom Backround Textures In The Scratch Website

### DISCUSSION THREAD

**Discussion by RisingGlitch** [ScratchAddons/ScratchAddons#4985] [Ideas]
So, I Know We Have Themes And Everything, But I Think It Would Be A Really Cool Idea Too Add The Support For Custom Background In The Scratch Website, But It Would Be Cool Idea If You Could Add The Ability To Customise Your Background So When You Are Looking Through Like The Main Page or Forums or just in general Browsing Through The Projects You Could Have A Background Of Your Choice Showing Up In The Screen! I Feel Like It Will Add A New Cool Feel To Scratch In General, And Just Be Really Cool!

In The Add-on You Can Choose Between Using Your Own Image From Your Computer Or An Image URL!

Example:  (Yes Not Only The Background Changed, But Just Mind The Background)
<img width="1783" alt="Screen Shot 2022-08-14 at 7 04 16 PM" src="https://user-images.githubusercontent.com/95631896/184559878-bc1c3811-64c9-49f2-9fcd-72073886cd62.png">

**Comment by GoofyAhhScratch** [comment]
you should also use gifs!
